package.json

```diff
@@ -3,9 +3,10 @@
   "private": true,
   "version": "1.0.0",
   "nodeVersions": {
-    "v10.x": "v10.5.0",
-    "v8.x": "v8.11.3",
-    "v6.x": "v6.14.2"
+    "v12.x": "v12.6.0",
```

Comment:
Missing update the test (`defines nodeVersions`) with the version `v12.x`.

Reply:
Sorry, could you explain more what do you mean?

Reply:
Okay, I see you mean `test.js`